testing/python/collect.py

```diff
@@ -531,6 +531,84 @@ def test2(self, x, y):
         assert colitems[2].name == 'test2[a-c]'
         assert colitems[3].name == 'test2[b-c]'
 
+    def test_parametrize_skipif(self, testdir):
```

Comment:
A lot of copy paste here, but I thought prudent to avoid using `parametrize` while writing tests for `parametrize`. :grin: